tests/e2e/arm_test.go

```diff
@@ -40,7 +40,7 @@ var _ = Describe("ARM image generation", Label("arm"), func() {
 			Expect(out).To(ContainSubstring("done"), out)
 			Expect(out).To(ContainSubstring("build-arm-image"), out)
 			Expect(err).ToNot(HaveOccurred())
-			_, err = os.Stat(filepath.Join(tempDir, "build/build/disk.img"))
+			_, err = os.Stat(filepath.Join(tempDir, "build/disk.img"))
```

Comment:
all this strings in here, should they  be a var? so we can change iot easily just in case?

Reply:
I will postpone this for a future cleanup. We got more things to do anyway (e.g. https://github.com/kairos-io/kairos/issues/3020).